File: example/scalalib/testing/6-test-group-stealing/foo/test/src/GroupX1.scala
package foo
import utest._
object GroupX1 extends TestSuite {
  def tests = Tests {
    test("test1") {
      val result = Foo.greet("Aether")
      assert(result == "Hello Aether")
      Thread.sleep(42)
      result
    }
    test("test2") {
      val result = Foo.greet("Boreas")
      assert(result == "Hello Boreas")
      Thread.sleep(18)
      result
    }
  }
}

File: example/kotlinlib/testing/6-test-group-stealing/foo/test/src/foo/GroupX1.kt
package foo

import org.junit.jupiter.api.Test

class GroupX1 : RandomTestsUtils() {
    @Test
    @Throws(Exception::class)
    fun test1() { testGreeting("Aether", 55) }
    
    @Test
    @Throws(Exception::class)
    fun test2() { testGreeting("Boreas", 43) }
} 